Rubberduck.Parsing/Symbols/Declaration.cs

```diff
@@ -295,22 +295,41 @@ public string DescriptionString
         {
             get
             {
+                string literalDescription;
+
                 var memberAttribute = _attributes.SingleOrDefault(a => a.Name == $"{IdentifierName}.VB_Description");
                 if (memberAttribute != null)
                 {
-                    return memberAttribute.Values.SingleOrDefault() ?? string.Empty;
+                    literalDescription = memberAttribute.Values.SingleOrDefault() ?? string.Empty;
+                    return CorrectlyFormatedDescription(literalDescription);
                 }
 
                 var moduleAttribute = _attributes.SingleOrDefault(a => a.Name == "VB_Description");
                 if (moduleAttribute != null)
                 {
-                    return moduleAttribute.Values.SingleOrDefault() ?? string.Empty;
+                    literalDescription = moduleAttribute.Values.SingleOrDefault() ?? string.Empty;
+                    return CorrectlyFormatedDescription(literalDescription);
                 }
 
                 return string.Empty;
             }
         }
 
+        private static string CorrectlyFormatedDescription(string literalDescription)
```

Comment:
could use a short doc-comment why exactly an outer set of quote marks isn't "correctly formatted"